src/IssueViz.UnitTests/Editor/QuickActions/QuickFixes/QuickFixActionsSourceTests.cs

```diff
@@ -151,7 +151,7 @@ public async Task HasSuggestedActionsAsync_NoIssuesWithQuickFixes_False()
         [TestMethod]
         public async Task HasSuggestedActionsAsync_HasIssuesWithQuickFixes_True()
         {
-            var issues = new[] { CreateIssueViz("quick-fix1") };
+            var issues = new[] { CreateIssueViz(Mock.Of<IQuickFix>()) };
```

Comment:
@ugras-ergun-sonarsource FYI, replaced to mock as the tests don't care about the actual type